src/content/en/showcase/2017/with.in.md

```diff
@@ -0,0 +1,331 @@
+project_path: /web/_project.yaml
```

Comment:
Please rename this file with-in.md or within.md or something similar, so not include an extra dot in the file name.